.circleci/config.yml

```diff
@@ -542,24 +541,15 @@ workflows:
       - build:
           requires:
             - dependencies
-      - codecov:
-          requires:
-            - build
       - test:
           requires:
             - dependencies
-      - racetest:
-          requires:
-            - dependencies
       - e2e-simple:
           requires:
             - build
       - e2e-bookinfo:
           requires:
             - build
-      - e2e-mixer:
```

Comment:
we should keep this one - do you have a link to the failures?

Reply:
We can move them around - it'll still be executed, just in a different cron. Look for the 'periodic' task for failures.